[wasm] Library tests failing during linking for AOT - `SIGKILL`

Build Browser wasm Linux Release LibraryTests_AOT fails randomly on CI

This might be caused by OOM as the wasm AOT builds are memory intensive.

CI main run fail - [build](https://dev.azure.com/dnceng-public/public/_build/results?buildId=106348&view=results), and [log](https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-heads-main-c02253fc001044a985/normal-System.Runtime.Tests/1/console.704a9d9e.log?helixlogtype=result).

PR fail https://github.com/dotnet/runtime/runs/10048986689

```
  Linking with emcc with -Oz -Wl,-O0 -Wl,-lto-O0. This may take a while ...
   "/datadisks/disk1/work/A19508CB/p/build/emsdk/upstream/bin/wasm-ld" @/datadisks/disk1/work/A19508CB/t/emscripten_idqz6a8k.rsp.utf-8
   "/datadisks/disk1/work/A19508CB/p/build/emsdk/upstream/bin/wasm-emscripten-finalize" -g --dyncalls-i64 --dwarf /datadisks/disk1/work/A19508CB/w/B4A90A00/e/wasm_build/obj/wasm/for-build/dotnet.wasm -o /datadisks/disk1/work/A19508CB/w/B4A90A00/e/wasm_build/obj/wasm/for-build/dotnet.wasm --detect-features
   "/datadisks/disk1/work/A19508CB/p/build/emsdk/node/14.18.2_64bit/bin/node" /datadisks/disk1/work/A19508CB/p/build/emsdk/upstream/emscripten/src/compiler.js /datadisks/disk1/work/A19508CB/t/tmplw9d5ysg.json
   "/datadisks/disk1/work/A19508CB/p/build/emsdk/upstream/bin/wasm-opt" --post-emscripten -Oz --low-memory-unused --zero-filled-memory --strip-producers /datadisks/disk1/work/A19508CB/w/B4A90A00/e/wasm_build/obj/wasm/for-build/dotnet.wasm -o /datadisks/disk1/work/A19508CB/w/B4A90A00/e/wasm_build/obj/wasm/for-build/dotnet.wasm -g --mvp-features --enable-mutable-globals
emcc : error : '/datadisks/disk1/work/A19508CB/p/build/emsdk/upstream/bin/wasm-opt --post-emscripten -Oz --low-memory-unused --zero-filled-memory --strip-producers /datadisks/disk1/work/A19508CB/w/B4A90A00/e/wasm_build/obj/wasm/for-build/dotnet.wasm -o /datadisks/disk1/work/A19508CB/w/B4A90A00/e/wasm_build/obj/wasm/for-build/dotnet.wasm -g --mvp-features --enable-mutable-globals' failed (received SIGKILL (-9)) [/datadisks/disk1/work/A19508CB/w/B4A90A00/e/publish/ProxyProjectForAOTOnHelix.proj]
/datadisks/disk1/work/A19508CB/p/build/wasm/WasmApp.Native.targets(431,5): error MSB3073: The command "emcc "@/datadisks/disk1/work/A19508CB/p/build/microsoft.netcore.app.runtime.browser-wasm/runtimes/browser-wasm/native/src/emcc-default.rsp" "@/datadisks/disk1/work/A19508CB/p/build/microsoft.netcore.app.runtime.browser-wasm/runtimes/browser-wasm/native/src/emcc-link.rsp" "@/datadisks/disk1/work/A19508CB/w/B4A90A00/e/wasm_build/obj/wasm/for-build/emcc-link.rsp"" exited with code 1. [/datadisks/disk1/work/A19508CB/w/B4A90A00/e/publish/ProxyProjectForAOTOnHelix.proj]
```

## Error Message
Fill the error message using [known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#how-to-fill-out-a-known-issue-error-message-section).

```json
{
  "ErrorMessage": "dotnet.wasm -g --mvp-features --enable-mutable-globals' failed (received SIGKILL (-9))",
  "BuildRetry": false
}
```

<!--Known issue error report start -->
### Report

|Build|Definition|Test|Pull Request|
|---|---|---|---|
|[122829](https://dev.azure.com/dnceng-public/public/_build/results?buildId=122829)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=122829&view=ms.vss-test-web.build-test-results-tab&runId=2639568&resultId=100001)|dotnet/runtime#80103|
|[112529](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112529)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112529&view=ms.vss-test-web.build-test-results-tab&runId=2542052&resultId=100001)|dotnet/runtime#79671|
|[113478](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113478)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113478&view=ms.vss-test-web.build-test-results-tab&runId=2486770&resultId=100001)|dotnet/runtime#79708|
|[110492](https://dev.azure.com/dnceng-public/public/_build/results?buildId=110492)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=110492&view=ms.vss-test-web.build-test-results-tab&runId=2452388&resultId=100001)|dotnet/runtime#79589|
|[115598](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115598)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115598&view=ms.vss-test-web.build-test-results-tab&runId=2446172&resultId=100001)|dotnet/runtime#79790|
|[115574](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115574)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115574&view=ms.vss-test-web.build-test-results-tab&runId=2445434&resultId=150635)||
|[115569](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115569)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115569&view=ms.vss-test-web.build-test-results-tab&runId=2444994&resultId=100001)||
|[115554](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115554)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115554&view=ms.vss-test-web.build-test-results-tab&runId=2444570&resultId=100001)|dotnet/runtime#79775|
|[115562](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115562)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115562&view=ms.vss-test-web.build-test-results-tab&runId=2444604&resultId=100001)|dotnet/runtime#79776|
|[115512](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115512)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115512&view=ms.vss-test-web.build-test-results-tab&runId=2443434&resultId=100001)|dotnet/runtime#79787|
|[115497](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115497)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115497&view=ms.vss-test-web.build-test-results-tab&runId=2442654&resultId=149499)||
|[115484](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115484)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115484&view=ms.vss-test-web.build-test-results-tab&runId=2442320&resultId=100001)||
|[115218](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115218)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115218&view=ms.vss-test-web.build-test-results-tab&runId=2442134&resultId=100001)|dotnet/runtime#79720|
|[115452](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115452)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115452&view=ms.vss-test-web.build-test-results-tab&runId=2441838&resultId=100001)|dotnet/runtime#79790|
|[115372](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115372)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115372&view=ms.vss-test-web.build-test-results-tab&runId=2440986&resultId=100001)|dotnet/runtime#79685|
|[115315](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115315)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115315&view=ms.vss-test-web.build-test-results-tab&runId=2438892&resultId=100001)|dotnet/runtime#79788|
|[115310](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115310)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115310&view=ms.vss-test-web.build-test-results-tab&runId=2438802&resultId=100001)|dotnet/runtime#79787|
|[115246](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115246)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115246&view=ms.vss-test-web.build-test-results-tab&runId=2437668&resultId=100001)||
|[115237](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115237)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115237&view=ms.vss-test-web.build-test-results-tab&runId=2437252&resultId=100001)|dotnet/runtime#79775|
|[115164](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115164)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115164&view=ms.vss-test-web.build-test-results-tab&runId=2436240&resultId=100001)|dotnet/runtime#79779|
|[115160](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115160)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115160&view=ms.vss-test-web.build-test-results-tab&runId=2436186&resultId=100001)|dotnet/runtime#79778|
|[115136](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115136)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115136&view=ms.vss-test-web.build-test-results-tab&runId=2436048&resultId=100001)|dotnet/runtime#79659|
|[114879](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114879)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114879&view=ms.vss-test-web.build-test-results-tab&runId=2436018&resultId=100001)|dotnet/runtime#79120|
|[114878](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114878)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114878&view=ms.vss-test-web.build-test-results-tab&runId=2435680&resultId=100001)||
|[114868](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114868)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114868&view=ms.vss-test-web.build-test-results-tab&runId=2435478&resultId=100001)|dotnet/runtime#79720|
|[115097](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115097)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115097&view=ms.vss-test-web.build-test-results-tab&runId=2435448&resultId=100001)|dotnet/runtime#79775|
|[115070](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115070)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115070&view=ms.vss-test-web.build-test-results-tab&runId=2435044&resultId=100001)|dotnet/runtime#79329|
|[115057](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115057)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115057&view=ms.vss-test-web.build-test-results-tab&runId=2434904&resultId=100001)|dotnet/runtime#78861|
|[115050](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115050)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=115050&view=ms.vss-test-web.build-test-results-tab&runId=2434800&resultId=100001)|dotnet/runtime#79169|
|[114983](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114983)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114983&view=ms.vss-test-web.build-test-results-tab&runId=2434258&resultId=100001)|dotnet/runtime#79544|
|[114974](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114974)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114974&view=ms.vss-test-web.build-test-results-tab&runId=2434062&resultId=100001)|dotnet/runtime#79659|
|[114929](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114929)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114929&view=ms.vss-test-web.build-test-results-tab&runId=2433136&resultId=149752)||
|[114790](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114790)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114790&view=ms.vss-test-web.build-test-results-tab&runId=2430676&resultId=100001)|dotnet/runtime#79763|
|[114763](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114763)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114763&view=ms.vss-test-web.build-test-results-tab&runId=2430314&resultId=100001)|dotnet/runtime#79659|
|[114659](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114659)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114659&view=ms.vss-test-web.build-test-results-tab&runId=2429312&resultId=100001)|dotnet/runtime#79759|
|[114347](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114347)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114347&view=ms.vss-test-web.build-test-results-tab&runId=2429274&resultId=100001)|dotnet/runtime#79613|
|[114629](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114629)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114629&view=ms.vss-test-web.build-test-results-tab&runId=2429074&resultId=100001)|dotnet/runtime#79544|
|[114569](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114569)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114569&view=ms.vss-test-web.build-test-results-tab&runId=2427160&resultId=100001)|dotnet/runtime#79046|
|[114521](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114521)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114521&view=ms.vss-test-web.build-test-results-tab&runId=2426238&resultId=100001)|dotnet/runtime#79754|
|[114507](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114507)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114507&view=ms.vss-test-web.build-test-results-tab&runId=2426142&resultId=100001)|dotnet/runtime#79685|
|[114428](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114428)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114428&view=ms.vss-test-web.build-test-results-tab&runId=2425460&resultId=100001)|dotnet/runtime#79448|
|[109567](https://dev.azure.com/dnceng-public/public/_build/results?buildId=109567)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=109567&view=ms.vss-test-web.build-test-results-tab&runId=2425334&resultId=100001)|dotnet/runtime#79215|
|[114378](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114378)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114378&view=ms.vss-test-web.build-test-results-tab&runId=2423614&resultId=100001)|dotnet/runtime#79639|
|[114323](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114323)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114323&view=ms.vss-test-web.build-test-results-tab&runId=2422210&resultId=100001)|dotnet/runtime#79744|
|[114261](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114261)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114261&view=ms.vss-test-web.build-test-results-tab&runId=2420776&resultId=150015)||
|[114247](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114247)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114247&view=ms.vss-test-web.build-test-results-tab&runId=2419530&resultId=100001)||
|[114173](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114173)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114173&view=ms.vss-test-web.build-test-results-tab&runId=2418112&resultId=100001)|dotnet/runtime#79737|
|[114122](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114122)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114122&view=ms.vss-test-web.build-test-results-tab&runId=2417206&resultId=100001)|dotnet/runtime#79720|
|[114106](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114106)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114106&view=ms.vss-test-web.build-test-results-tab&runId=2416892&resultId=100001)|dotnet/runtime#79730|
|[114071](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114071)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=114071&view=ms.vss-test-web.build-test-results-tab&runId=2416290&resultId=100001)|dotnet/runtime#78742|
|[112612](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112612)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112612&view=ms.vss-test-web.build-test-results-tab&runId=2415214&resultId=100001)|dotnet/runtime#79677|
|[113764](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113764)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113764&view=ms.vss-test-web.build-test-results-tab&runId=2412584&resultId=149772)||
|[113770](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113770)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113770&view=ms.vss-test-web.build-test-results-tab&runId=2412576&resultId=100001)|dotnet/runtime#79721|
|[113740](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113740)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113740&view=ms.vss-test-web.build-test-results-tab&runId=2411856&resultId=100001)|dotnet/runtime#79719|
|[113728](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113728)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113728&view=ms.vss-test-web.build-test-results-tab&runId=2411762&resultId=100001)|dotnet/runtime#79718|
|[113700](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113700)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113700&view=ms.vss-test-web.build-test-results-tab&runId=2410966&resultId=100001)|dotnet/runtime#79716|
|[113696](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113696)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113696&view=ms.vss-test-web.build-test-results-tab&runId=2410992&resultId=100001)||
|[113704](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113704)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113704&view=ms.vss-test-web.build-test-results-tab&runId=2411030&resultId=100001)|dotnet/runtime#79717|
|[113263](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113263)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113263&view=ms.vss-test-web.build-test-results-tab&runId=2407588&resultId=100001)|dotnet/runtime#77399|
|[113497](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113497)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113497&view=ms.vss-test-web.build-test-results-tab&runId=2407500&resultId=100001)|dotnet/runtime#79686|
|[113458](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113458)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113458&view=ms.vss-test-web.build-test-results-tab&runId=2406542&resultId=100001)|dotnet/runtime#79613|
|[113268](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113268)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113268&view=ms.vss-test-web.build-test-results-tab&runId=2405910&resultId=100001)|dotnet/runtime#79659|
|[113381](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113381)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113381&view=ms.vss-test-web.build-test-results-tab&runId=2405260&resultId=100001)|dotnet/runtime#79685|
|[113324](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113324)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113324&view=ms.vss-test-web.build-test-results-tab&runId=2403576&resultId=100001)|dotnet/runtime#79145|
|[113298](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113298)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113298&view=ms.vss-test-web.build-test-results-tab&runId=2401674&resultId=100001)|dotnet/runtime#79651|
|[113283](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113283)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113283&view=ms.vss-test-web.build-test-results-tab&runId=2401322&resultId=100001)|dotnet/runtime#79698|
|[113181](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113181)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113181&view=ms.vss-test-web.build-test-results-tab&runId=2399606&resultId=150025)||
|[112336](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112336)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112336&view=ms.vss-test-web.build-test-results-tab&runId=2399368&resultId=100001)|dotnet/runtime#79651|
|[113158](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113158)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113158&view=ms.vss-test-web.build-test-results-tab&runId=2398826&resultId=100001)||
|[113155](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113155)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113155&view=ms.vss-test-web.build-test-results-tab&runId=2398814&resultId=147824)|dotnet/runtime#79339|
|[112809](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112809)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112809&view=ms.vss-test-web.build-test-results-tab&runId=2398680&resultId=100001)|dotnet/runtime#79686|
|[113093](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113093)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=113093&view=ms.vss-test-web.build-test-results-tab&runId=2397600&resultId=100001)|dotnet/runtime#78862|
|[111861](https://dev.azure.com/dnceng-public/public/_build/results?buildId=111861)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=111861&view=ms.vss-test-web.build-test-results-tab&runId=2394950&resultId=100001)|dotnet/runtime#77399|
|[112813](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112813)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112813&view=ms.vss-test-web.build-test-results-tab&runId=2394598&resultId=100001)|dotnet/runtime#76500|
|[112793](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112793)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112793&view=ms.vss-test-web.build-test-results-tab&runId=2394260&resultId=100001)|dotnet/runtime#78862|
|[112670](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112670)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112670&view=ms.vss-test-web.build-test-results-tab&runId=2392308&resultId=100001)|dotnet/runtime#79544|
|[112589](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112589)|dotnet/runtime|[normal-System.Runtime.Serialization.Xml.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112589&view=ms.vss-test-web.build-test-results-tab&runId=2391754&resultId=147940)||
|[112637](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112637)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112637&view=ms.vss-test-web.build-test-results-tab&runId=2391862&resultId=100001)|dotnet/runtime#79678|
|[112600](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112600)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112600&view=ms.vss-test-web.build-test-results-tab&runId=2391706&resultId=100001)|dotnet/runtime#79675|
|[112557](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112557)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112557&view=ms.vss-test-web.build-test-results-tab&runId=2391338&resultId=146249)|dotnet/runtime#79672|
|[112315](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112315)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112315&view=ms.vss-test-web.build-test-results-tab&runId=2391040&resultId=100001)|dotnet/runtime#79659|
|[112564](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112564)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112564&view=ms.vss-test-web.build-test-results-tab&runId=2390910&resultId=100001)|dotnet/runtime#78860|
|[112560](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112560)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112560&view=ms.vss-test-web.build-test-results-tab&runId=2390904&resultId=100001)|dotnet/runtime#79669|
|[112551](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112551)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112551&view=ms.vss-test-web.build-test-results-tab&runId=2390812&resultId=100001)||
|[112553](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112553)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112553&view=ms.vss-test-web.build-test-results-tab&runId=2390772&resultId=100001)|dotnet/runtime#79672|
|[112378](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112378)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112378&view=ms.vss-test-web.build-test-results-tab&runId=2387394&resultId=100001)|dotnet/runtime#79324|
|[112295](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112295)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112295&view=ms.vss-test-web.build-test-results-tab&runId=2385782&resultId=100001)|dotnet/runtime#76500|
|[112303](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112303)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112303&view=ms.vss-test-web.build-test-results-tab&runId=2385676&resultId=100001)|dotnet/runtime#79652|
|[112291](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112291)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112291&view=ms.vss-test-web.build-test-results-tab&runId=2385594&resultId=100001)|dotnet/runtime#79657|
|[112269](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112269)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112269&view=ms.vss-test-web.build-test-results-tab&runId=2385420&resultId=100001)|dotnet/runtime#79472|
|[112286](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112286)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112286&view=ms.vss-test-web.build-test-results-tab&runId=2385534&resultId=100001)|dotnet/runtime#79448|
|[112209](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112209)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112209&view=ms.vss-test-web.build-test-results-tab&runId=2384646&resultId=100001)|dotnet/runtime#79644|
|[112152](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112152)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112152&view=ms.vss-test-web.build-test-results-tab&runId=2383462&resultId=100001)|dotnet/runtime#78557|
|[112066](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112066)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112066&view=ms.vss-test-web.build-test-results-tab&runId=2382534&resultId=100001)|dotnet/runtime#79645|
|[111843](https://dev.azure.com/dnceng-public/public/_build/results?buildId=111843)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=111843&view=ms.vss-test-web.build-test-results-tab&runId=2382080&resultId=100001)|dotnet/runtime#79334|
|[112020](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112020)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112020&view=ms.vss-test-web.build-test-results-tab&runId=2379880&resultId=100001)|dotnet/runtime#79644|
|[112013](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112013)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112013&view=ms.vss-test-web.build-test-results-tab&runId=2379628&resultId=100001)|dotnet/runtime#79640|
|[112009](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112009)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=112009&view=ms.vss-test-web.build-test-results-tab&runId=2379360&resultId=100001)|dotnet/runtime#79639|
|[111952](https://dev.azure.com/dnceng-public/public/_build/results?buildId=111952)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=111952&view=ms.vss-test-web.build-test-results-tab&runId=2378522&resultId=100001)|dotnet/runtime#79046|
|[111961](https://dev.azure.com/dnceng-public/public/_build/results?buildId=111961)|dotnet/runtime|[normal-System.Runtime.Tests.WorkItemExecution](https://dev.azure.com/dnceng-public/public/_build/results?buildId=111961&view=ms.vss-test-web.build-test-results-tab&runId=2378536&resultId=100001)|dotnet/runtime#79339|
Displaying 100 of 139 results
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|0|1|139|
<!--Known issue error report end -->

--- a/eng/testing/tests.wasm.targets
+++ b/eng/testing/tests.wasm.targets
@@ -176,7 +176,8 @@
     <PropertyGroup>
       <_MainAssemblyPath Condition="'%(WasmAssembliesToBundle.FileName)' == $(AssemblyName) and '%(WasmAssembliesToBundle.Extension)' == '.dll'">%(WasmAssembliesToBundle.Identity)</_MainAssemblyPath>
       <RuntimeConfigFilePath>$([System.IO.Path]::ChangeExtension($(_MainAssemblyPath), '.runtimeconfig.json'))</RuntimeConfigFilePath>
-      <EmccLinkOptimizationFlag Condition="'$(EmccLinkOptimizationFlag)' == ''">-Oz -Wl,-O0 -Wl,-lto-O0</EmccLinkOptimizationFlag>
+      <EmccLinkOptimizationFlag Condition="'$(EmccLinkOptimizationFlag)' == ''">-O1 -Wl,-O0 -Wl,-lto-O0</EmccLinkOptimizationFlag>
+      <WasmNativeStrip>true</WasmNativeStrip>
     </PropertyGroup>
 
     <Error Text="Item WasmAssembliesToBundle is empty. This is likely an authoring error." Condition="@(WasmAssembliesToBundle->Count()) == 0" />

